assets/src/editor/blocks/landing-page-hero/index.js

```diff
@@ -86,12 +86,12 @@ export const settings = {
 		buttonText: {
 			type: 'string',
 			source: 'html',
-			selector: '.hero__call-to-action',
+			selector: '.hero__cta-button',
 		},
 		buttonLink: {
 			type: 'string',
 			source: 'attribute',
-			selector: '.hero__call-to-action',
+			selector: 'two .hero__cta-button',
```

Comment:
Same here... and also, what is this `two` doing here? Maybe testing code left in accidentally...?

Reply:
Definitely Than, both were left as a mistake. Thanks for catching this!